### PR TITLE
Value too long error

### PR DESCRIPTION
**Pull request by dbrgn**
Some workarounds to alleviate message length problems reported through support.

### CODE REVIEW

File: src/directives/compose_area.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ export default [
                         // Check for max length
                         const byteLength = (new TextEncoder().encode(text)).length;
                         if (byteLength > scope.maxTextLength) {
-                            const pieces: string[] = stringService.byteChunk(text, scope.maxTextLength, 50);
+                            const pieces: string[] = stringService.byteChunk(text, scope.maxTextLength * 0.85, 50);
```
Comment (lgrahl) [inline src/directives/compose_area.ts]: Eh?

Comment (dbrgn) [reply src/directives/compose_area.ts]: Did you read the commit message?

Comment (dbrgn) [reply src/directives/compose_area.ts]: (A `Math.floor` call is missing though...)

Comment (lgrahl) [reply src/directives/compose_area.ts]: Umm, ok, but since it's a workaround, perhaps add a comment?

Comment (dbrgn) [reply src/directives/compose_area.ts]: Ok, fair point.

**Pull request merged**